library/psa_crypto.c

```diff
@@ -5513,7 +5618,9 @@ static psa_status_t psa_hkdf_input( psa_hkdf_key_derivation_t *hkdf,
             return( PSA_ERROR_INVALID_ARGUMENT );
     }
 }
+#endif /* MBEDTLS_PSA_BUILTIN_ALG_HKDF */
 
+#if defined(MBEDTLS_PSA_BUILTIN_ALG_TLS12_PRF)
```

Comment:
This function and the next one should be guarded by
defined(MBEDTLS_PSA_BUILTIN_ALG_TLS12_PRF) || \                             
    defined(MBEDTLS_PSA_BUILTIN_ALG_TLS12_PSK_TO_MS) 
as those functions are called by TLS12_PRF and TLS12_PSK_TO_MS code (see psa_tls12_prf_input()).